tests/NSubstitute.Acceptance.Specs/FormattingCallsWhenThrowingReceivedCallsExceptions.cs

```diff
@@ -85,14 +85,14 @@ protected override void ExpectedCall()
             [Test]
             public void Should_report_the_method_we_were_expecting()
             {
-                ExceptionMessageContains(ExpectedCallMessagePrefix + "SampleMethod(\"string\", 1, List<String>)");
+                ExceptionMessageContains(ExpectedCallMessagePrefix + @"SampleMethod(""string"", 1, [""a"", ""b""])");
             }
 
             [Test]
             public void Should_indicate_which_args_are_different()
             {
-                ExceptionMessageContains("SampleMethod(*\"different\"*, 1, *List<String>*)");
-                ExceptionMessageContains("SampleMethod(\"string\", *7*, *List<String>*)");
+                ExceptionMessageContains("SampleMethod(*\"different\"*, 1, [**])");
```

Comment:
Maybe should also have a test here for `Sample.SampleMethod("another", 1, new List<string> { "a", "c", "d" );` to show which args get highlighted?

Reply:
Good catch, this currently breaks with a `NullReferenceException`.

Reply:
Fixed.